ui/apps/platform/src/Components/CompoundSearchFilter/utils/searchFilterConfig.test.ts

```diff
@@ -1,45 +1,91 @@
-import { imageSearchFilterConfig } from '../types';
-import { getFilteredConfig } from './searchFilterConfig';
+import { getImageAttributes } from '../attributes/image';
+import { createSearchFilterConfig } from './searchFilterConfig';
 
 describe('searchFilterConfig', () => {
-    describe('getFilteredConfig', () => {
+    describe('createSearchFilterConfig', () => {
         it('should get the image config with selected attributes', () => {
-            const result = getFilteredConfig(imageSearchFilterConfig, ['Name', 'Tag', 'Label']);
+            const result = createSearchFilterConfig([
+                {
+                    displayName: 'Image',
+                    searchCategory: 'IMAGES',
+                    attributes: getImageAttributes(['Name', 'Tag', 'Label']),
+                },
+            ]);
```

Comment:
See previous comment regarding whether or not `createSearchFilterConfig` is needed. Regardless though:
1. TY for including tests! 👊
2. Could these helpers benefit from tests that receive a) an empty array b) an array with length > 1

Reply:
Thanks for the suggestion, yes looks like we won't need that function anymore. Will be removing these 👍🏼